feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -159,6 +162,14 @@ class PreviewViewModel @AssistedInject constructor(
         }
     }
 
+    private fun getUseCaseMode(previewMode: PreviewMode): CameraUseCase.UseCaseMode {
```

Comment:
nit: When you have an exhaustive conversion function like this, you can make it an extension function:

```
private fun PreviewMode.toUseCaseMode() = when (this) {
    is PreviewMode.ExternalImageCaptureMode -> CameraUseCase.UseCaseMode.IMAGE_ONLY
    is PreviewMode.ExternalVideoCaptureMode -> CameraUseCase.UseCaseMode.VIDEO_ONLY
    is PreviewMode.StandardMode -> CameraUseCase.UseCaseMode.STANDARD
}
```